### DISCUSSION TITLE
Spatially variable genes across multiple samples

### DISCUSSION THREAD

**Discussion by franciscogrisanti** [satijalab/seurat#4204] [Q&A] [answered]
Hi guys!

I was wondering if anyone had come across the problem of dealing with spatial transcriptomics data and asking/characterizing spatially variable genes across multiple samples? Thanks in advance! 

Thanks in advance!

Francisco

**Comment by simoncmo** [comment, marked as answer]
Hi franciscogrisanti,

If I understand correctly you have an integrated object of multiple spatial transcriptomics samples, and would like to use FindSpatiallyVariableFeatures on all the sample in the object all at once?

My understanding is that, since of the function takes location information into account, and the spatial relationships between samples are unknown, it can only be done sample by sample. The `image` parameter in the therefore was used to tell the function from which sample to extract location information.
My approach has been running the function separately and combine them. Depends on the propose, it could an intersect or union of the SpatiallyVariableFeatures. 

Of course this is my approach, would love to hear other's or Seurat staff's opinion on this.

Thanks,
Simon

**Reply by franciscogrisanti** [reply]
Hi @simoncmo! 

Thanks for the response. My approach has been the same as yours, I just thought they might be a more statistically robust way to do this currently available. I'm thinking alignment will play a big role!

Best,

Francisco

**Reply by simoncmo** [reply]
Yeah to me that sounds like an effort that require some tool development works, so hope we can see such thing developed by Satija lab or other groups in the near future. 

thanks,
Simon
